src/components/Header.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { ImUser } from 'react-icons/im';
 
 const Header = () => (
   <header>
-    <h1>bookstore</h1>
+    <h1>Bookstore CMS</h1>
     <nav>
       <Link to="/">Books</Link>
       <Link to="/categories">Categories</Link>
     </nav>
-    <span>Log in</span>
+    <span>
+      <ImUser />
+    </span>
   </header>
 );
 
```

Comment:
- [ ] Please try to meet with the design provided, I see you are almost there as well as closer to the design but there are few implementations remaining.

- [ ] Original 
![image](https://user-images.githubusercontent.com/55635977/155837087-d9cea4ec-9da4-4965-93b0-5023cbf65f71.png)

- [ ] Yours
![image](https://user-images.githubusercontent.com/55635977/155837114-03c21225-e97f-44b3-9c91-1cce9494b34f.png)

Reply:
# Done
<img width="1440" alt="Captura de Pantalla 2022-02-26 a la(s) 4 10 23" src="https://user-images.githubusercontent.com/36294012/155839187-0e12e9bd-b116-4856-be5c-cab7f4d0e56b.png">